Goobi/src/de/sub/goobi/helper/tasks/ExportNewspaperBatchTask.java

```diff
@@ -771,6 +771,48 @@ private static Integer positionByRank(List<DocStruct> siblings, String metadataT
         return result;
     }
 
+    /**
+     * Returns the index of the issue to insert between its siblings depending
+     * on its rank. A return value of {@code null} will indicate that no
+     * position could be determined which will cause
+     * {@link DocStruct#addChild(Integer, DocStruct)} to simply append the new
+     * child at the end.
+     *
+     * @param siblings
+     *            brothers and sisters of the child to add
+     * @param rank
+     *            rank of the child to insert
+     * @return the index position to insert the child
+     */
+    private static Integer positionByRank(List<DocStruct> siblings, Integer rank) {
+        if (siblings != null || rank != null) {
```

Comment:
Is your condition `||` correct? If `siblings` is `null` and `rank` not `null` then it would create a null pointer exception in the next line.

Reply:
Yes, that was wrong, of course. Thank you. Of course it should be called `&&`.